miso-web/src/main/webapp/scripts/list_kits.js

```diff
@@ -49,7 +49,9 @@ ListTarget.kit = {
       "sTitle": "Version",
       "include": true,
       "iSortPriority": 0,
-      "mData": "version"
+      "mData": function(data){
+        return data.version? data.version : null;
```

Comment:
Should probably change the condition to `data.hasOwnProperty('version')` in-case of version set to `0`

Reply:
Fixed!